themes/jaeger-docs/assets/sass/pages.sass

```diff
@@ -52,12 +52,13 @@
       position: fixed
       bottom: 0%
       top: 9rem
+      left: 3rem 
```

Comment:
I tried to play with this one and didn't understand what it affects, seems turning it off makes the sidebar actually larger

Reply:
I added this to fix some of the strangeness in the scroll bar, but it depends on the browser you test with. If you don't think we need it I can remove it, just let me know.

Reply:
no, it's good, I see what it's doing.